ansible_base/rbac/migrations/0001_initial.py

```diff
@@ -67,6 +67,7 @@ class Migration(migrations.Migration):
                 ('modified', models.DateTimeField(auto_now=True, help_text='The date/time this resource was created')),
             ],
```

Comment:
Part of my reason for prioritizing this work now is because the AWX & EDA branches are not merged yet. So it's only used in feature branches... as of today.

Also, this isn't a DB schema change so we're find to make this change without disruption.